python/pyproject.toml

```diff
@@ -72,31 +72,17 @@ warn_unused_ignores = false
 show_error_codes = true
 ignore_missing_imports = true
 
-[tool.isort]
-line_length = 120
-src_paths = ["demo", "dolfinx", "test"]
-known_first_party = ["basix", "dolfinx", "ffcx", "ufl"]
-known_third_party = ["gmsh", "numba", "numpy", "pytest", "pyvista"]
-known_mpi = ["mpi4py", "petsc4py"]
-sections = [
-      "FUTURE",
-      "STDLIB",
-      "MPI",
-      "THIRDPARTY",
-      "FIRSTPARTY",
-      "LOCALFOLDER",
-]
 
 [tool.ruff]
-line-length = 120
+line-length = 100
```

Comment:
Was this on purpose?

(asking because the very first bullet in the PR says about some lines too long errors)

Reply:
Yes, 100 is considered the 'maximum' permissible length, see e.g. https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length